openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py

```diff
@@ -21,6 +21,10 @@ class ValidateExpectedFiles(pyblish.api.InstancePlugin):
 
     def process(self, instance):
         self.instance = instance
+        # TODO: Find a better way to check whether a job has been submitted
```

Comment:
@fabiaserra, I couldn't find an alternative solution. Maybe @kalisp or @iLLiCiTiT can help. Since `render_job_id` is included in `instance.data` even if it's None, your proposed approach seems best. To make changes, we'd need to modify the upstream code in `openpype\plugins\publish\collect_rendered_files.py` and remove the mandatory call of the key.